lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,21 @@
+module Overcommit::Hook::PrePush
+  # Invokes Git LFS command that uploads files tracked by Git LFS to the LFS storage
+  #
+  # @see https://git-lfs.github.com/
+  class GitLfs < Base
+    def run
+      result = execute(['command', '-v', 'git-lfs'])
```

Comment:
We have a helper for just this purpose: [`in_path?`](https://github.com/brigade/overcommit/blob/336c495d23befc7139810f2d9d58e42f8494313e/lib/overcommit/hook/base.rb#L89)